public/css/style.css

```diff
@@ -107,7 +108,7 @@
 
 /* Plot all D3 SVGs to center of their divs */
 .plotbox {
-    text-align: center;
+    text-align: left;
```

Comment:
LOL, look at the comment above, and then at [this image](https://twitter.com/nzkoz/status/538892801941848064). =D